CITATION.cff

```diff
@@ -0,0 +1,27 @@
+cff-version: 1.2.0
```

Comment:
What's this file used for?

Reply:
this file adds a button in the right-bar, like this image: 
<img width="505" alt="Bildschirmfoto 2025-05-20 um 14 27 41" src="https://github.com/user-attachments/assets/d52948eb-d6da-4069-8a7b-d63f497a4223" />